tools/example_showcase.sh

```diff
@@ -3,16 +3,16 @@
 duration='2'
 function wait_seconds() { perl -e 'alarm shift; exec @ARGV' "$@"; }
```

Comment:
This should probably not be calling into perl, and this entire script should instead just be a `cargo xtask` style command (for cross platform compatibility).

However, that's out of scope for this PR - this is an improvement.